client/extensions/woocommerce/app/settings/payments/payment-method-row.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import ListTd from '../../../components/list/list-td';
+
+class PaymentMethodRow extends Component {
```

Comment:
Where do we stand on using `class` components vs `function(){}` components?

In `woocommerce-services` we have agreed to always use function components when possible (i.e. when the component doesn't have `state`), but I see that `wp-calypso` doesn't have a convention in that regard.